kubernetes/helm/pinot/templates/broker/service-external.yaml

```diff
@@ -23,19 +23,17 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ template "pinot.broker.external" . }}
+  {{- if .Values.broker.service.gcpInternalLB }}
+  annotations:
+    cloud.google.com/load-balancer-type: Internal
```

Comment:
if check should go into annotations?

Reply:
annotations:
    {{- if .Values.broker.service.gcpInternalLB }}
    cloud.google.com/load-balancer-type: Internal
    {{- end. }}